### PR TITLE
LBA: Twinsen's Quest

### PR DESCRIPTION
**Pull request by Erant**
# Description

Add Little Big Adventure: Twinsen's Quest.

## Type(s) 

- [x] Game page addition

## Verification 
Have you

- [x] Followed the [contribution guidelines?](https://github.com/Whisky-App/whisky-book?tab=readme-ov-file#please-read-all-steps-before-contributing)
- [x] Ran `./scripts/generate.mjs`?
- [x] Ran `./scripts/lint.mjs`?
- [x] Checked spelling and grammar?

### CODE REVIEW

**Review by mattmaniak — CHANGES_REQUESTED** [review]
Please include updated `src/SUMMARY.md` with a new game title in your PR.

File: src/game-support/lba-twinsens-quest.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+The function `Core_IsAPISupported` in the aforementioned DLL needs to be patched to always return false. This prevents calling an API that isn't implemented.
+
+# Patching
+
+For the DLL with SHA256 `946478c5e8f61cbe75c6d70e3aabe2766126605cba45ddb579a5ef38fb198e98` three bytes at offset 0x600 need to be changed to `b0 00 c3`. The resulting file should have a SHA256 of `954fa6b2960b2567797f061ec11131d026c1c640e5dfb8e9d4d2bae593151d36`
+
+# Issues
+
+Game is playable through the end, with all cutscenes playing correctly and no graphical corruption observed. Playing with a controller sometimes makes Twinsen perform an action continuously. This issue only occurs when loading or starting a game and is resolved by quitting and restarting the game.
```
Comment (mattmaniak) [inline src/game-support/lba-twinsens-quest.md:20]: Please add a Steam game template with an ID of 2318070 at the cery bottom of the file. You can find examples in other titles there. It helps users to locate the game on Steam.

**Review by mattmaniak — CHANGES_REQUESTED** [review]
Add Steam ID template as mentioned in a previous comment.

File: src/game-support/lba-twinsens-quest.md (hunk 1 above)
Comment (Erant) [reply src/game-support/lba-twinsens-quest.md:20]: I'll add it, though I didn't acquire the game through Steam and given the binary patching required wasn't willing to guess the GOG version exactly matched the Steam one.

Comment (mattmaniak) [reply src/game-support/lba-twinsens-quest.md:20]: Sure. I will have to think about handling such cases as this Steam ID requirement is convenient from front-end side but it is not clear wheter to include it or not for non-Steam-tested games. Currently I decided to add it to keep it consistent with other titles. Thanks for your understanding and pushes.

**Review by mattmaniak — APPROVED** [review]

**Pull request merged**